concepts/language-model.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model tries to predict the next word given the already produced history.
+For example, given the text span `the man with` a language model's output could be `a: 0.4, the: 0.3, my: 0.1, his: 0.05, ...`.
+This would mean that the article `a` has a 40% probability of being the continuation and is the most likely.
+
+It is possible to do *decoding*, which means repeatedly taking e.g. the most probable output, adding it to the text and so on.
+This way a language model can generate new text.
+
+# Models
+
+There are many different ways in which language models are created.
+
+## N-gram Maximum Likelihood Estimate
```

Comment:
We generally use sentence case, not title case, in titles.

Reply:
Didn't know that, changed.